docs/SetupNNIDeveloperEnvironment.md

```diff
@@ -24,6 +24,8 @@ to install the dependent tools for the environment
 
 Run the command
 ```
+sudo apt-get install -y python3-setuptools
```

Comment:
It assumes that the python3 and pip3 are the prerequisites of running NNI, see the above text in this document.